src/main/java/page/mentors/EditMentorsDetailsPage.java

```diff
@@ -0,0 +1,100 @@
+package page.mentors;
+
+import constants.Locators;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import page.base.Page;
+
+public class EditMentorsDetailsPage extends Page implements Locators.EditMentorsDetailsPage {
+    public EditMentorsDetailsPage(WebDriver driver) {
```

Comment:
We agreed that we all will import 'Locators class', so you must import class? not implements
For example:
"import static constants.Locators.Courses.*;"